### PR TITLE
refactor(AnalyticalTable): migrate from react-jss to css modules

### PR DESCRIPTION
**Pull request by Lukas742**

### CODE REVIEW

File: packages/main/src/components/AnalyticalTable/AnalyticalTable.module.css [hunk 1]
```diff
@@ -0,0 +1,312 @@
...
+  position: relative;
+}
+
+.overlay {
+  position: absolute;
+  z-index: 1;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
```
Comment (MarcusNotheis) [inline packages/main/src/components/AnalyticalTable/AnalyticalTable.module.css:36]: could be replaced with inset: 0?

File: packages/main/src/components/AnalyticalTable/AnalyticalTable.module.css [hunk 2]
```diff
@@ -0,0 +1,312 @@
...
+  }
+
+  &[data-is-selected]:hover {
+    background-color: var(--sapList_Hover_SelectionBackground);
+  }
+}
+
+.trActive {
+  cursor: pointer;
+}
```
Comment (MarcusNotheis) [inline packages/main/src/components/AnalyticalTable/AnalyticalTable.module.css:167]: this used to be `.trActive:hover` - intended change?

**Review by MarcusNotheis — APPROVED** [review]

File: packages/main/src/components/AnalyticalTable/AnalyticalTable.module.css (hunk 1 above)
Comment (Lukas742) [reply packages/main/src/components/AnalyticalTable/AnalyticalTable.module.css:36]: I kept it like this as it shouldn't make a difference for different writing-modes, but this way we're save if these styles are overwritten somewhere else.

File: packages/main/src/components/AnalyticalTable/AnalyticalTable.module.css (hunk 2 above)
Comment (Lukas742) [reply packages/main/src/components/AnalyticalTable/AnalyticalTable.module.css:167]: Imo it doesn't make sense setting this style only on hover as the cursor is anyway only visible when hovered.

**Pull request merged**